server/api/router.go

```diff
@@ -134,6 +134,8 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	registerFunc(apiRouter, "/schedulers", schedulerHandler.CreateScheduler, setMethods("POST"))
 	registerFunc(apiRouter, "/schedulers/{name}", schedulerHandler.DeleteScheduler, setMethods("DELETE"))
 	registerFunc(apiRouter, "/schedulers/{name}", schedulerHandler.PauseOrResumeScheduler, setMethods("POST"))
+	registerFunc(apiRouter, "/schedulers/{name}/diagnose", schedulerHandler.DiagnoseScheduler, setMethods("POST"))
```

Comment:
How about using `/diagnose` as the prefix? We can pass the specified scheduler as a parameter.